CONTRIBUTING.md

```diff
@@ -65,6 +69,9 @@ $ sbt publishLocal
 $ sbt manual/previewSite
 ~~~
 
-## General recommendations
+## Publish a release
 
-Test and document each added feature.
+- Check that the `changelog.md` file is up to date (in the `manual` project)
+- Draft a new release on [GitHub](https://github.com/scalacenter/scalajs-bundler/releases/new)
```

Comment:
Why is this necessary? When we push a tag from git, GitHub automatically creates a GitHub release with the tag number, and compresses the repository in a zip that you can download.

Reply:
You’re right either way works (push a git tag or publish a tag from GitHub). Since we are not currently using GitHub releases to show our changelog we don’t really care about manually creating a GitHub release, so I should rather mention the way that consists in pushing a tag.

Reply:
Yeah, just say "push a tag" with github (and maybe inline the command to do so). The CI will take care of everything.